src/odb/src/lefout/lefout.cpp

```diff
@@ -897,57 +908,59 @@ void lefout::writeVia(dbTechVia* via)
     writeBoxes(boxes, "    ");
   } else {
     std::string rname = rule->getName();
-    fprintf(_out, "\n    VIARULE %s \n", rname.c_str());
+    fmt::print(_out, "\n    VIARULE {} \n", rname.c_str());
 
     dbViaParams P;
     via->getViaParams(P);
 
-    fprintf(_out,
-            " + CUTSIZE %.11g %.11g ",
-            lefdist(P.getXCutSize()),
-            lefdist(P.getYCutSize()));
+    fmt::print(_out,
+               " + CUTSIZE {:.11g} {:.11g} ",
+               lefdist(P.getXCutSize()),
+               lefdist(P.getYCutSize()));
     std::string top = P.getTopLayer()->getName();
     std::string bot = P.getBottomLayer()->getName();
     std::string cut = P.getCutLayer()->getName();
-    fprintf(_out, " + LAYERS %s %s %s ", bot.c_str(), cut.c_str(), top.c_str());
-    fprintf(_out,
-            " + CUTSPACING %.11g %.11g ",
-            lefdist(P.getXCutSpacing()),
-            lefdist(P.getYCutSpacing()));
-    fprintf(_out,
-            " + ENCLOSURE %.11g %.11g %.11g %.11g ",
-            lefdist(P.getXBottomEnclosure()),
-            lefdist(P.getYBottomEnclosure()),
-            lefdist(P.getXTopEnclosure()),
-            lefdist(P.getYTopEnclosure()));
+    fmt::print(
+        _out, " + LAYERS {} {} {} ", bot.c_str(), cut.c_str(), top.c_str());
+    fmt::print(_out,
+               " + CUTSPACING {:.11g} {:.11g} ",
+               lefdist(P.getXCutSpacing()),
+               lefdist(P.getYCutSpacing()));
+    fmt::print(_out,
+               " + ENCLOSURE {:.11g} {:.11g} {:.11g} {:.11g} ",
```

Comment:
While not an issue with the translation, it seems there should be new lines in these statements.